### PR TITLE
Github Link Opens In New Tab

### LINKED ISSUE #2159: GitHub link open in same page
### What would you like to share?

When we click on GitHub (present on navigation bar) the repo is opening in same tab. We should open new tab for opening github repo.
image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by LinuxGamer**
## Fixes Issues
Closes #2159 

## Proposed Changes
- Makes Github link open in new tab by setting target to "_blank"

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots
N/A

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2190"></a>

### CODE REVIEW

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ export default function Navbar() {
                     href="https://github.com/EddieHubCommunity/LinkFree"
                     className="text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
                     aria-current="page"
+		             target="_blank"
+		             rel="noreferrer"
```
Comment (eddiejaoude) [inline components/navbar/Navbar.js:64]: I think the indentation is incorrect?

Comment (kumarsonsoff3) [reply components/navbar/Navbar.js:64]: I guess it should be `noopener`. 

Since `noreferrer` will impact the analytics and tracking!

**Comment by LinuxGamer** [comment]
Should be fixed the indentation now. I have tested and it opens. noopener shows that's it's an outdated rel thingy and wouldn't allow me to commit.

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**